lib/core/routing/routes_manager.dart

```diff
@@ -57,8 +72,15 @@ class RouteGenerator {
           ),
         );
 
-
-
+      case Routes.onboardingRoute:
+        return MaterialPageRoute(
+          builder: (_) => MultiBlocProvider(
```

Comment:
why MultiBlocProvider?

Reply:
In anticipation of potential future requirements to add a Bloc, MultiBlocProvider is currently utilized within the routes. This approach is believed not to impact application performance. Would it be more efficient to remove MultiBlocProvider and implement BlocProvider directly?